src/components/calcite-fab/calcite-fab.scss

```diff
@@ -1,7 +1,7 @@
 :host {
   @extend %component-host;
```

Comment:
you could remove this and put in the individual properties of the class:

```
  box-sizing: border-box;
  * {
    box-sizing: border-box;
  }
  background-color: var(--calcite-ui-foreground-1);
  color: var(--calcite-ui-text-2);
  font-size: var(--calcite-font-size--1);
```

Honestly I'm not sure why some of these are on here as they don't do anything...

Reply:
the spacing vars aren't even used here? I think you could remove that one.

Reply:
I can try removing them and see if screener has any changes?